pages/common/nikto.md

```diff
@@ -0,0 +1,27 @@
+# nikto
+
+> Web server scanner which performs comprehensive tests against web servers for multiple items, including over 6700 potentially dangerous files/programs.
```

Comment:
Let us remove the last part of the sentence. It is a bit unnecessary for a heading to contain such extraneous info.